awx/resource_organization_instance_groups.go

```diff
@@ -0,0 +1,95 @@
+/*
+*TBD*
+
+# Example Usage
+
+```hcl
+
+	resource "awx_organization_galaxy_credential" "baseconfig" {
+	  organization_id = awx_organization.baseconfig.id
+	  credential_id   = awx_credential_machine.pi_connection.id
+	}
+
+```
+*/
+package awx
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	awx "stash.ovh.net/corobmet/goawx/client"
+)
+
+func resourceOrganizationsInstanceGroups() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceOrganizationsInstanceGroupsCreate,
+		DeleteContext: resourceOrganizationsInstanceGroupsDelete,
+		ReadContext:   resourceOrganizationsInstanceGroupsRead,
+
+		Schema: map[string]*schema.Schema{
+
+			"organization_id": {
+				Type:     schema.TypeInt,
+				Required: true,
+				ForceNew: true,
+			},
+			"instance_groups_id": {
+				Type:     schema.TypeInt,
+				Required: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func resourceOrganizationsInstanceGroupsCreate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	var diags diag.Diagnostics
+	client := m.(*awx.AWX)
+	awxService := client.OrganizationsService
+	OrganizationID := d.Get("organization_id").(int)
+	_, err := awxService.GetOrganizationsByID(OrganizationID, make(map[string]string))
+	if err != nil {
+		return buildDiagNotFoundFail("organization", OrganizationID, err)
+	}
+
+	result, err := awxService.AssociateInstanceGroups(OrganizationID, map[string]interface{}{
```

Comment:
This PR seems to depends on https://github.com/denouche/goawx/pull/32 
Am I right?

Reply:
if so, your PR should depends on denouche/goawx v0.21.0, and not an internal repository from OVH